clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -1861,13 +1867,23 @@ def CallOp : CIR_CallOpBase<"call", [NoRegionArguments]> {
   let arguments = commonArgs;
 
   let builders = [OpBuilder<(ins "mlir::SymbolRefAttr":$callee,
-                                 "mlir::Type":$resType,
-                                 "mlir::ValueRange":$operands), [{
+                                "mlir::Type":$resType,
+                                "mlir::ValueRange":$operands),
+                            [{
       $_state.addOperands(operands);
       $_state.addAttribute("callee", callee);
```

Comment:
The incubator doesn't do this if callee is null. Does that matter?

Reply:
Since this builder is for building a direct call I believe it's more proper to have an assertion here that ensure `callee` is non-empty.